compilers/concrete-compiler/compiler/include/concretelang/Dialect/FHELinalg/IR/FHELinalgOps.td

```diff
@@ -1415,5 +1415,107 @@ def FHELinalg_ReinterpretPrecisionEintOp: FHELinalg_Op<"reinterpret_precision",
     let hasVerifier = 1;
 }
 
+def FHELinalg_FancyIndexOp : FHELinalg_Op<"fancy_index", [Pure]> {
```

Comment:
Yes, we've discussed those operations with @andidr, and decided to have our own operations because conversion patterns for those operations is not on the upstream, and they are more generic. So if we were to support them entirely, we would need to spend more time on the task. Let's keep it like this for the time being and refactor it later if they got proper support on the upstream.

Reply:
Make sense, it's what I expected. :+1: